pyhealth/medcode/inner_map.py

```diff
@@ -33,7 +33,6 @@ def __init__(
     ):
         # abstractmethod prevents initialization of this class
         self.vocabulary = vocabulary
-        self.refresh_cache = refresh_cache
```

Comment:
this class variable wasn't used anywhere; there isn't one in the `CrossMap` class

Reply:
good catch.